tooling/create-tauri-app/src/recipes/solid.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019-2021 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+import { join } from 'path'
+import { shell } from '../shell'
+import { Recipe } from '../types/recipe'
+
+const solid: Recipe = {
+  descriptiveName: {
+    name: 'Solid (https://github.com/solidjs/solid)',
+    value: 'solid'
+  },
+  shortName: 'solid',
+  extraNpmDevDependencies: [],
+  extraNpmDependencies: [],
+  extraQuestions: ({ ci }) => {
+    return [
+      {
+        type: 'list',
+        name: 'template',
+        message: 'Which Solid template would you like to use? (Read more at https://github.com/solidjs/templates)',
```

Comment:
With the requested change above, we should remove the link from here.

Reply:
Absolutely. Yeah I see Svelte is doing the same thing. Done!